packages/components/tests/dummy/app/templates/components/badge-count.hbs

```diff
@@ -0,0 +1,118 @@
+{{page-title "BadgeCount component"}}
+
+<h2 class="dummy-h2">BadgeCount</h2>
+
+<section>
+  <h3 class="dummy-h3" id="bc-overview"><a href="#bc-overview" class="dummy-link-section">§</a> Overview</h3>
```

Comment:
now that the pages are separated, no more need of different IDs, we can use the "standard/conventional" ones.
```suggestion
  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
```